src/ClientResolver.php

```diff
@@ -683,12 +683,15 @@ public static function _apply_endpoint_provider($value, array &$args)
 
             $args['endpoint'] = $result['endpoint'];
 
-            if (
-                empty($args['config']['signature_version'])
-                && isset($result['signatureVersion'])
-            ) {
-                $args['config']['signature_version']
-                    = $result['signatureVersion'];
+            if (empty($args['config']['signature_version'])) {
```

Comment:
Shouldn't this already be handled by the defaultProvider?  Unless `bearer` is modeled differently than other auth types I don't think we'll need this check